### PR TITLE
added support for FreeBSD

### PR DESCRIPTION
**Pull request by alxwr**
Tested on Ubuntu 15.10 and FreeBSD 10.
I'm planning to stick with this module and provide updates for FreeBSD if needed. :-)

Thanks for sharing!

### CODE REVIEW

File: manifests/service.pp [hunk 1]
```diff
@@ -21,23 +21,25 @@
     default => false,
   }
 
-  if $openntp::service_restart == $openntp::params::service_restart {
-    # The restart script does not properly check if the service is actually stopped; hence, the start command can fail
-    # to start the service because it is still running. This script checks the status and issues the start command as
-    # soon as the service is actually stopped.
+  if $openntp::service_restart != undef {
+    if $openntp::service_restart == $openntp::params::service_restart {
```
Comment (tohuwabohu) [inline manifests/service.pp]: Would it make sense to merge both if conditions into one if block?

Comment (alxwr) [inline manifests/service.pp]: I separated the two if blocks for other variants of openntp::service_restart to come, but the two blocks can certainly be merged.

Comment (tohuwabohu) [inline manifests/service.pp]: hmm k. Looks like YAGNI to me but I'm happy to roll with it for the moment.

**Comment by tohuwabohu** [comment]
I've released 2.3.0 to the forge. Thanks for your contribution.

**Comment by alxwr** [comment]
Thanks or merging!

**Pull request merged**